### PR TITLE
Feature/components

### PR DESCRIPTION
**Pull request by over-geek**
### BookStore: Initialize project with components

In this milestone, I:
- setup client-side routing using [react router v6](https://reactrouter.com/en/main)
- Created a Header component inside of the `/components` directory
- Created a Button component inside of the `/components` directory
- Created a Books component inside of the `/components` directory to render books
- Created a Form component inside of the `/components` directory to add a new book
- Created a button to delete the book

### CODE REVIEW

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
.

**Comment by over-geek** [comment]
Hello @tufoinnkuo10 your comments are hidden. Can you review the code again? I do not have any failing linters

File: README.md [hunk 1]
```diff
@@ -1,70 +1,207 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```
Comment (tufoinnkuo10) [inline README.md]: - [ ] nice work so far. Please your issue link is broken, kind fix that to keep your README file professional.

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @over-geek 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Gitflow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ components well organized

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by Kamba56 — APPROVED** [review]
Hi @over-geek, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
## Highlights
- [x] Great job using Create react app to create bookstore!👏👏👏
- [x] Great job adding books and form components!👏👏👏
- [x] !👏👏👏

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**